CHANGELOG.md

```diff
@@ -1,35 +1,136 @@
-* Add new "Geology/Mineral occurrences" layer from GEUS.
-* Update `layer_list.csv` to include new column indicating if each layer is
+- Resolve `Font "Helvetica" font not available on system` warnings
+- Update `layer_list.csv` to include new column indicating if each layer is
   stored on disk. Internet-required layers take the value `False`.
-* Add new "Glaciology/Ice sheet velocity" layers:
-  * "GrIMP/Annual ice sheet velocity vectors 2021"
-  * "GrIMP/Annual ice sheet velocity magnitude 2021 (200m)"
-  * "GrIMP/Annual ice sheet velocity x component 2021 (200m)"
-  * "GrIMP/Annual ice sheet velocity y component 2021 (200m)"
-* Improve symbology for "Regional climate models/RACMO model output/Annual mean
+- Establish a more explicit/readable pattern for defining the order of QGreenland's
+  layers and groups in the Layers Panel
+- Configure gradient style legends, add new style validations for gradient legends
+- Fix palletted styles
+- Publish versioned Docker images to DockerHub and GHCR for improved reproducibility
+
+
+## :warning: Breaking changes
+
+- Minimum supported version of QGIS is now `v3.28`.
+- QGreenland Custom is no longer supported.
+
+
+## Documentation
+
+- Restructured documentation for improved readability
+- Added documentation of QGreenland future plans under next NSF grant
+- Improved recommended citations for QGreenland
+
+
+## Layers
+
+### New
+
+- Add new "Places/Populated places" layer, replacing the "Places/Towns and
+  settlements" layer.
+- Add new "Internet-required data/Geological Map (1:500 000)" layer from GEUS.
+- Add new "Geology/Mineral occurrences" layer from GEUS.
+- Add new "Glaciology/Ice sheet velocity" layers:
+    - "GrIMP/Annual ice sheet velocity vectors 2021"
+    - "GrIMP/Annual ice sheet velocity magnitude 2021 (200m)"
+    - "GrIMP/Annual ice sheet velocity x component 2021 (200m)"
+    - "GrIMP/Annual ice sheet velocity y component 2021 (200m)"
+- Add "Geology/Tectonic plates" polygon layer. This layer accompanies the
+  existing "Geology/Tectonic plate boundaries" polyline layer from the same
+  dataset.
+- Add new WMS layers from the Danish Agency for Data Supply and Infrastructure
+  (SDFI):
+    - "Internet-required data/Topographic map of Greenland"
+    - "Internet-required data/Satellite orthophoto mosaic (10-0.2m)"
+- Add new "Internet-required data/Blue Marble shaded relief and Bathymetry
+  (500m)" WMS layer from NASA Global Imagery Browse Services (GIBS)
+- Add new "Places/" layers from Asiaq/NunaGIS:
+    - Buildings
+    - Roads
+- Add new "Biology/Vegetation/Vegetation classification map (1km)" layer from
+  the Raster Circumpolar Arctic Vegetation Map produced by Raynolds et al.,
+  2019.
+
+
+### Updated
+
+- Fix descriptions of "Hydrology" outlets and basins layers.
+- Rename "Places/Place names database" -> "Places/Points of interest". Towns and
+  settlements are now excluded from this layer (in favor of the more-up-to-date
+  "Towns" and "Settlements" layers.
```

Comment:
```suggestion
  "Populated places" layer.
```

We merged "Towns" and "Settlements" into one "Populated places" layer.

Reply:
Woops, got it one place but not all the places :laughing: